### PR TITLE
feat: dymension

### PR DESCRIPTION
**Pull request by BabyScope**
changed name of dymension

### CODE REVIEW

File: cookbook/specs/dymension.json [hunk 1]
```diff
@@ -0,0 +1,1939 @@
...
+                    {
+                      "expected_value": "0x44c"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "expected_value": ""
```
Comment (omerlavanet) [inline cookbook/specs/dymension.json:1798]: why an empty expected value?

File: cookbook/specs/dymension.json [hunk 2]
```diff
@@ -0,0 +1,1939 @@
...
+                    },
+                    {
+                      "extension": "archive",
+                      "expected_value": ""
```
Comment (omerlavanet) [inline cookbook/specs/dymension.json:1802]: same q

**Review by omerlavanet — CHANGES_REQUESTED** [review]

File: cookbook/specs/dymension.json (hunk 1 above)
Comment (BabyScope) [reply cookbook/specs/dymension.json]: this was a mistake.

File: cookbook/specs/dymension.json (hunk 2 above)
Comment (BabyScope) [reply cookbook/specs/dymension.json]: solved it

**Comment by Yaroms** [comment]
currently not needed, thanks for the work

**Pull request closed without merging**